package.json

```
@@ -123,7 +123,7 @@
   "packageManager": "pnpm@8.6.0",
   "engines": {
     "node": ">=14",
-    "pnpm": ">=8"
+    "pnpm": ">=8.6.0"
```

```suggestion
    "pnpm": "^8.6.2"
```

这样子吧，我预感到 pnpm v9 不远了😅